docs/product/integrations/notification-incidents/slack/index.mdx

```diff
@@ -56,7 +56,7 @@ Set up an alert rule by going to **Alerts** and clicking "Create Alert". From he
 
 #### Issue Alerts
 
-In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown and then select your workspace and channel:
+In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown and then select your workspace and channel. You can optionally select any tags to include in the alert as well as other text in the "notes" section. Mentions are supported in the "notes" section - to @ a user, include their user id like `@<USER_ID>`.
```

Comment:
```suggestion
In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown and then select your workspace and channel. You can optionally select any supported tags to include in the alert, as well as other text in the "notes" section. Mentions are supported in the "notes" section - to @ a user, include their user id like `@<USER_ID>`.
```

Reply:
Is there a list of supported tags anywhere that's documented?

Reply:
can users have custom tags too? like pass them when they raise an error? otherwise i see sections for tags for each platform but not a singular place for all possible tags

Reply:
Ohh okay, I see. Sorry I thought this was limited.